tests/integ/test_stored_procedure.py

```diff
@@ -16,6 +16,14 @@
 
 import pytest
 
+from tests.integ.datasource.test_oracledb import (
+    ORACLEDB_TEST_EXTERNAL_ACCESS_INTEGRATION,
+)
+from tests.parameters import ORACLEDB_CONNECTION_PARAMETERS
```

Comment:
these two imports may cause failure in our snowfort tests, I don't remember seeing `tests.parameters ` in our snowfort setup.

you can move them into your test method and add a skip in non GH env to your test case